### PR TITLE
Jonathan Marx & Ivan Bird

### PR DESCRIPTION
**Pull request by JonnyMarx98**

### CODE REVIEW

File: Biodome.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    Biodome.set_alpha(alpha)
+
+alpha = 250
+alpha0 = 250
+alpha1 = 175
+alpha2 = 125
+alpha3 = 75
+alpha4 = 50
+alpha5 = 20
+
```
Comment (Herro-Sama) [inline Biodome.py:36]: Could use more comments to explain what you are doing.

File: Biodome.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            elif BackColour == BLACK:
+                BackColour = random.choice([WHITE, WHITE, BLUE, colour1, colour2, colour3])
+            elif BackColour == colour1:
+                BackColour = random.choice([WHITE, BLACK, BLUE, RED, colour2, colour3])
+            elif BackColour == colour2:
+                BackColour = random.choice([WHITE, BLACK, WHITE, colour1, BLUE, colour3])
+            elif BackColour == colour3:
+                BackColour = random.choice([WHITE, WHITE, BLUE, colour1, colour2, BLACK])
+            screen.fill((BackColour))
+
```
Comment (Herro-Sama) [inline Biodome.py:109]: Could these if statements be a part of a class?

File: Biodome.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
+import pygame, sys, random, time, math, timeit
+from pygame.locals import *
+
+pygame.init()
+clock = pygame.time.Clock()
+
+WIDTH = 720
```
Comment (hannahlu) [inline Biodome.py:7]: Could change width/height variables to lowercase.

File: Bridge.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+            if event.type == QUIT:
+                pygame.quit()
+                sys.exit()
+            #esc key exits
+            if event.type == pygame.KEYDOWN:
+                if event.key == pygame.K_ESCAPE:
+                    exit()
+            if event.type == pygame.KEYDOWN:
+                if event.key == pygame.K_1:
+                    drawBridge()
```
Comment (hannahlu) [inline Bridge.py:36]: Try adding comments to explain each section of code.

File: Wreckage.py [hunk 5]
```diff
@@ -0,0 +1,99 @@
+
+import pygame, sys, time, random, math
+from pygame.locals import *
+
+pygame.init()
+pygame.mixer.init()
+
+
+
+Width = 900
```
Comment (hannahlu) [inline Wreckage.py:10]: Change variables to all uppercase.

File: Wreckage.py [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+    del px_array
+    window.blit(picture, (0, 0))
+    updates += 1
+    darkenAll(updates)
+    alienTorch()
+
+
+
+
+    pygame.display.update()
```
Comment (hannahlu) [inline Wreckage.py:99]: Could add comments to code and get rid of unnecessary spacing.

File: Biodome.py [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+                BackColour = random.choice([WHITE, BLACK, WHITE, colour1, colour2, colour3])
+            elif BackColour == BLACK:
+                BackColour = random.choice([WHITE, WHITE, BLUE, colour1, colour2, colour3])
+            elif BackColour == colour1:
+                BackColour = random.choice([WHITE, BLACK, BLUE, RED, colour2, colour3])
+            elif BackColour == colour2:
+                BackColour = random.choice([WHITE, BLACK, WHITE, colour1, BLUE, colour3])
+            elif BackColour == colour3:
+                BackColour = random.choice([WHITE, WHITE, BLUE, colour1, colour2, BLACK])
+            screen.fill((BackColour))
```
Comment (Klumz) [inline Biodome.py]: Align comments to the side for improved readability.

Comment (Klumz) [inline Biodome.py:108]: Align comments to the side for improved readability.

File: Biodome.py [hunk 8]
```diff
@@ -0,0 +1,112 @@
+import pygame, sys, random, time, math, timeit
```
Comment (jack-maber) [inline Biodome.py:1]: Packages should be imported on separate lines according to PEP-8.

**Pull request closed without merging**